supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/model/MemorySize.scala

```diff
@@ -0,0 +1,46 @@
+package cromwell.backend.impl.jes.model
+
+import wdl4s.parser.MemoryUnit
+
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+import scalaz.Scalaz._
+
+object MemorySize {
+  val memoryPattern = """(\d+(?:\.\d+)?)\s*(\w+)""".r
+
+  def parse(unparsed: String): Try[MemorySize] = {
+    unparsed match {
+      case memoryPattern(amountString, unitString) =>
+        val amount = amountString.parseDouble leftMap {
+          _.getMessage
+        } toValidationNel
+        val unit = MemoryUnit.values find {
+          _.suffixes.contains(unitString)
+        } match {
+          case Some(s) => s.successNel
+          case None => s"$unitString is an invalid memory unit".failureNel
+        }
+        (amount |@| unit) { (a, u) => new MemorySize(a, u) } match {
+          case scalaz.Success(memorySize) => Success(memorySize)
+          case scalaz.Failure(nel) => Failure(new UnsupportedOperationException(nel.list.mkString("\n")))
+        }
+      case _ => Failure(new UnsupportedOperationException(s"$unparsed should be of the form 'X Unit' where X is a number, e.g. 8 GB"))
+    }
+  }
+}
+
+case class MemorySize(amount: Double, unit: MemoryUnit) {
```

Comment:
I'm very confused about why this is in `cromwell.backend.impl.jes.model`  Seems to me like we could (and probably should) support memory sizes for local jobs too!